2021-02-24 13:34:15,344 INFO    MainThread:17474 [wandb_setup.py:_flush():70] setting env: {}
2021-02-24 13:34:15,344 INFO    MainThread:17474 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-24 13:34:15,345 INFO    MainThread:17474 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-24 13:34:15,345 INFO    MainThread:17474 [wandb_setup.py:_flush():70] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f216e06c730>
2021-02-24 13:34:15,345 INFO    MainThread:17474 [wandb_init.py:_log_setup():319] Logging user logs to /home/johannbs/Desktop/IceCube_GNN/wandb/run-20210224_133415-3sncdmtm/logs/debug.log
2021-02-24 13:34:15,345 INFO    MainThread:17474 [wandb_init.py:_log_setup():320] Logging internal logs to /home/johannbs/Desktop/IceCube_GNN/wandb/run-20210224_133415-3sncdmtm/logs/debug-internal.log
2021-02-24 13:34:15,345 INFO    MainThread:17474 [wandb_init.py:init():351] calling init triggers
2021-02-24 13:34:15,345 INFO    MainThread:17474 [wandb_init.py:init():356] wandb.init called with sweep_config: {}
config: {}
2021-02-24 13:34:15,346 INFO    MainThread:17474 [wandb_init.py:init():404] starting backend
2021-02-24 13:34:15,363 INFO    MainThread:17474 [backend.py:ensure_launched():81] starting backend process...
2021-02-24 13:34:15,370 INFO    MainThread:17474 [backend.py:ensure_launched():85] started backend process with pid: 17504
2021-02-24 13:34:15,372 INFO    MainThread:17474 [wandb_init.py:init():413] backend started and connected
2021-02-24 13:34:15,373 INFO    MainThread:17474 [wandb_init.py:init():436] updated telemetry
2021-02-24 13:34:15,374 INFO    MainThread:17474 [wandb_init.py:init():459] communicating current version
2021-02-24 13:34:16,134 INFO    MainThread:17474 [wandb_init.py:init():464] got version response upgrade_message: "wandb version 0.10.20 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-02-24 13:34:16,135 INFO    MainThread:17474 [wandb_init.py:init():472] communicating run to backend with 30 second timeout
2021-02-24 13:34:16,371 INFO    MainThread:17474 [wandb_init.py:init():495] starting run threads in backend
2021-02-24 13:34:17,840 INFO    MainThread:17474 [wandb_run.py:_console_start():1411] atexit reg
2021-02-24 13:34:17,840 INFO    MainThread:17474 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.WRAP
2021-02-24 13:34:17,840 INFO    MainThread:17474 [wandb_run.py:_redirect():1309] Wrapping output streams.
2021-02-24 13:34:17,840 INFO    MainThread:17474 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-24 13:34:17,840 INFO    MainThread:17474 [wandb_init.py:init():518] run started, returning control to user process
2021-02-24 13:34:18,952 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set hidden_states = 64 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,952 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb hidden_states 64 None
2021-02-24 13:34:18,952 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set forward = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,953 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb forward False None
2021-02-24 13:34:18,953 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set dropout = 0.25 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,953 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb dropout 0.25 None
2021-02-24 13:34:18,953 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set learning_rate = 0.0001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,953 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb learning_rate 0.0001 None
2021-02-24 13:34:18,954 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set batch_size = 1024 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,954 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb batch_size 1024 None
2021-02-24 13:34:18,954 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set loss_func = likelihood_covariant_unitvectors - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,954 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb loss_func likelihood_covariant_unitvectors None
2021-02-24 13:34:18,954 INFO    MainThread:17474 [wandb_config.py:__setitem__():142] config set n_neighbors = 6 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f2165d2ea00>>
2021-02-24 13:34:18,955 INFO    MainThread:17474 [wandb_run.py:_config_callback():663] config_cb n_neighbors 6 None
2021-02-24 15:53:06,433 INFO    MainThread:17474 [wandb_run.py:finish():1068] finishing run johannbs/likelihood_angle/3sncdmtm
2021-02-24 15:53:06,449 INFO    MainThread:17474 [wandb_init.py:_jupyter_teardown():250] cleaning up jupyter logic
2021-02-24 15:53:06,451 INFO    MainThread:17474 [jupyter.py:save_history():208] not saving jupyter history
